src/FakeItEasy/Core/FakeManager.cs

```diff
@@ -66,8 +66,9 @@ internal FakeManager(Type fakeObjectType, object proxy, IFakeManagerAccessor fak
         /// <summary>
         /// Gets the faked proxy object.
         /// </summary>
-        [SuppressMessage("Microsoft.Naming", "CA1716:IdentifiersShouldNotMatchKeywords", MessageId = "Object", Justification = "Renaming this property would be a breaking change.")]

```

Comment:
I think we could remove all suppression attributes and use `pragma warning disable` instead.

Reply:
I glossed over this comment the first time around. You mean _all_ `SuppressMessage`s!

Reply:
It was just a remark in passing. I don't intend to change all of them now, but we could. It's more concise, and just as readable (as long as we include the warning description)